src/main/kotlin/blackjack/domain/OwnCards.kt

```diff
@@ -0,0 +1,23 @@
+package blackjack.domain
+
+class OwnCards(private val _cards: MutableSet<Card> = buildSet { repeat(Draw.FIRST_DRAW_COUNT) { add(Card()) } }.toMutableSet()) {
+
+    val cards: Set<Card>
+        get() = _cards
+
+    fun sumCardNumber() = _cards.sumOf { if (it.cardNumber.isSmall) it.cardNumber.value % 10 else it.cardNumber.value }
```

Comment:
```suggestion
    fun sumCardNumber() = _cards.sumOf {
        if (it.cardNumber.isSmall) it.cardNumber.value % 10
        else it.cardNumber.value
    }
```

나중을 생각하여, 한 줄이 아닌 여러 칸으로 표현하는 것도 나쁘지 않아보이네요~